### PR TITLE
move ts dto to theia project

### PR DESCRIPTION
**Pull request by riuvshin**
move ts dto to theia project

### CODE REVIEW

**Review by evidolob — APPROVED** [review]

**Comment by benoitf** [comment]
-1, DTO are che objects

**Review by benoitf — CHANGES_REQUESTED** [review]
DTO are Eclipse Che objects

**Comment by benoitf** [comment]
@riuvshin : why is breaking release process ?

**Comment by riuvshin** [comment]
@benoitf it is looking for artifacts that are not published yet. So it has to be started after release and artifacts publishing is done.  So I will update PR to not execute that on maven build, and we will execute it from the tag when needed.

**Comment by benoitf** [comment]
fine as workaround

**Review by benoitf — APPROVED** [review]
workaround seems fine (to execute it after release)

**Comment by riuvshin** [comment]
@benoitf are you OK to move it to che-theia? I will replace pom.xml with `build.sh` that will build it in docker so it will not be as maven module

**Comment by benoitf** [comment]
no, it should not go to che-theia
it has nothing related to theia at all
che-theia is not a garbage repository.

I'm fine that it's not part of maven but part of build.sh

**Review by benoitf — CHANGES_REQUESTED** [review]
waiting the changes that only remove it from maven scope

**Comment by riuvshin** [comment]
OK

**Comment by riuvshin** [comment]
@benoitf @evidolob I've pushed changes

**Review by evidolob — APPROVED** [review]

File: typescript-dto/build.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+docker run -i --rm -v "$HOME/.m2:/root/.m2" \
+                   -v "$(pwd)/dto-pom.xml:/usr/src/mymaven/pom.xml" \
+                   -w /usr/src/mymaven maven:3.3-jdk-8 \
+                    /bin/bash -c "mvn -q -U -DskipTests=true -Dfindbugs.skip=true -Dskip-validate-sources install \
+                    && cat target/dts-dto-typescript.d.ts" >> index.d.ts
+
+CHE_VERSION=$(mvn -q -Dexec.executable=echo -Dexec.args='${project.version}' --non-recursive exec:exec -f ../pom.xml)
+
+docker build -t eclipse-che-ts-api --build-arg CHE_VERSION=${CHE_VERSION} --build-arg NPM_AUTH_TOKEN=${NPM_AUTH_TOKEN} .
```
Comment (evidolob) [inline typescript-dto/build.sh]: New line :)

**Review by benoitf — APPROVED** [review]
thx @riuvshin

File: typescript-dto/build.sh (hunk 1 above)
Comment (riuvshin) [reply typescript-dto/build.sh]: done

**Pull request merged**